rsconf/package_data/rsiviz/nginx.conf.jinja

```diff
@@ -1,17 +1,21 @@
 client_max_body_size 200m;
```

Comment:
I don't think this should be at this scope.

Reply:
38327fc